pages/ext/discord.vue

```diff
@@ -0,0 +1,234 @@
+                            $t('participateTrack')
+                        }}</span>
+                    </template>
+                </i18n>
+            </div>
+            <div class="section">
+                <core-h1 :title="$t('roleListHeader')"></core-h1>
+                <p class="paragraph-content">{{ $t('roleListDescription') }}</p>
+                <div v-for="role in $t('roleList')" :key="role.name">
+                    <p
+                        class="paragraph-title font-bold text-base md:text-lg lg:text-lg"
+                    >
+                        {{ role.name }}
+                    </p>
+                    <p class="paragraph-content">{{ role.description }}</p>
+                </div>
+            </div>
+            <div class="section">
+                <core-h1 :title="$t('channelListHeader')"></core-h1>
+                <div
+                    v-for="channelGroup in $t('channelGroup')"
+                    :key="channelGroup.type"
+                >
+                    <p
+                        class="paragraph-title font-bold text-base md:text-lg lg:text-lg"
+                    >
+                        {{ channelGroup.type }}
+                    </p>
+                    <div
+                        v-for="channel in channelGroup.channels"
+                        :key="channel.name"
+                        class="mb-3"
+                    >
+                        <p class="bold-title">{{ channel.name }}</p>
+                        <p class="paragraph-content">
+                            {{ $t('role') }}
+                            <span class="underline highlight">
+                                {{ channel.role }}
+                            </span>
+                        </p>
+                        <p class="paragraph-content">
+                            {{ channel.description }}
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="section">
+                <core-h1 :title="$t('appendix')"></core-h1>
+                <p class="paragraph-title">
+                    {{ $t('appendixStep1.title') }}
+                </p>
+                <i18n path="appendixStep1.content" tag="p">
+                    <template #link>
+                        <ext-link
+                            href="https://discord.com/download"
+                            highlight
+                            underline
+                            >{{ $t('appendixStep1.linkText') }}</ext-link
+                        >
+                    </template>
+                </i18n>
+                <p class="paragraph-title">
+                    {{ $t('appendixStep2.title') }}
+                </p>
+                <p class="paragraph-content">
+                    {{ $t('appendixStep2.content') }}
+                </p>
+                <img
+                    src="https://tw.pycon.org/2020/static/pycontw-2020/assets/discord-0.jpg"
```

Comment:
Not sure if the URL will remain the same after we archiving the website. Consider storing the picture in [pycontw/pycontw.asset](https://github.com/pycontw/pycontw.asset)